CompressImagesFunction/CompressImages.cs

```diff
@@ -83,6 +85,17 @@ public static bool Run(CompressimagesParameters parameters, ILogger logger)
                 return false;
             }
 
+            if (repoConfiguration.CompressWiki && !parameters.CloneUrl.Contains(".wiki.gi"))
```

Comment:
i feel like we should check this setting first before we look at the schedule setting. What do you think?

Reply:
Aaah, of course.. Otherwise we might miss compressing wiki just because it's not time for the code repo.

Will fix!